synapse/storage/schema/main/delta/73/01remove_unstable_private_read_receipts.sql

```diff
@@ -0,0 +1,17 @@
+/* Copyright 2022 The Matrix.org Foundation C.I.C
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+INSERT INTO background_updates (update_name, progress_json)
+    VALUES ('remove_unstable_private_read_receipts', '{}');
```

Comment:
Part of me thinks we shouldn't bother with a background update here and just run the queries on start-up TBH. Much less machinery and the queries *should* be fast.